src/router.js

```diff
@@ -1,20 +1,19 @@
 const express = require("express");
 const router = express.Router();
-const guardianListing = require("./api/guardian")
+const guardianListing = require("./api/guardian");
 const bodyParser = require("body-parser");
 
 router.get("/", (req, res) => {
-  res.render('home');
-})
-
+  res.render("home");
+});
 
 router.post("/articles", (req, res) => {
-query = req.body.search;
-    guardianListing(query, (results) => {
-      const articles = results.response.results
-        res.status(200).render('articles', { articles });
-      })
-})
+  query = req.body.search;
+  guardianListing(query, 1, results => {
+    const articles = results;
```

Comment:
why do you need this variable just pass the results to res.render

Reply:
yes, it has to have the same name when we pass it

Reply:
because it's called articles in the handlebars view, it has to match